_includes/core/footer.html

```diff
@@ -0,0 +1,20 @@
+<footer>
+  <div class="inner-footer">
+    <p>© 2015 Microsoft, Inc.</p>
+    <p>
+      <a href="https://twitter.com/microsoft" target="_blank"><span class="fa fa-stack fa-lg">
+        <span class="fa fa-square-o fa-stack-2x"></span>
+        <span class="fa fa-twitter fa-stack-1x"></span>
+      </span></a>
+      <a href="https://www.github.com/microsoft" target="_blank"><span class="fa fa-stack fa-lg">
+        <span class="fa fa-square-o fa-stack-2x"></span>
+        <span class="fa fa-github fa-stack-1x"></span>
+      </span></a>
+      <a href="https://www.facebook.com/Microsoft" target="_blank"><span class="fa fa-stack fa-lg">
```

Comment:
Well - that's a good point.  I should probably set up a proper Facebook presence for the .NET Foundation...